### ISSUE TITLE
Avoid using a variable named "long" in zshcomp

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5332]
|[<img alt="pitrou's avatar" src="https://avatars.githubusercontent.com/u/1721820?s=50" width="50" height="50">](https://github.com/pitrou)| @pitrou reported|
|-|-|
|Trac ID|trac#5332|
|Type|enhancement|
|Created|2011-10-22 17:50:26Z|
|Branch|https://github.com/twisted/twisted/tree/rename-long-5332|

2to3 replaces some of the occurrences (not all) with "int".
Patch attached.

Attachments:

* [zshcomp_long.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/d3e/d3efaa5acc98e8b4cf417af35296733a177541c6/9669aadfe32c664febae451518fdf6674c2c3665.patch) (10396 bytes) - added by antoine on 2011-10-22 17:50:32Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5332 5332
type__enhancement enhancement
reporter__antoine antoine
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_rename_long_5332 branches/rename-long-5332
branch_author__teratorn teratorn
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1319305826000000 1319305826000000
changetime__1319327375000000 1319327375000000
version__None None
owner__teratorn teratorn
cc__antoine cc__teratorn@...
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)|@teratorn set owner to @teratorn|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_3"></a>|@teratorn commented|
|-|-|

(In [32958]) Branching to 'rename-long-5332'

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_4"></a>|@teratorn removed owner|
|-|-|

This patch looks good - applied to branch.

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_5"></a>|@teratorn commented|
|-|-|

I've made similiar changes in twisted/python/_shellcomp.py

The only changes in this branch should be a renaming of variables called 'long' to 'longname'.

[build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/rename-long-5332). Ready for review.

**Comment by twisted-trac** [comment]
|[<img alt="pitrou's avatar" src="https://avatars.githubusercontent.com/u/1721820?s=50" width="50" height="50">](https://github.com/pitrou)<a name="note_6"></a>|@pitrou set owner to @teratorn|
|-|-|

"The changes lack a topfiles entry", he said after insistant help from the tutorial.
Otherwise, this looks good enough.

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_7"></a>|@teratorn set owner to @pitrou|
|-|-|

Fixed now. Thanks.

**Comment by twisted-trac** [comment]
|[<img alt="pitrou's avatar" src="https://avatars.githubusercontent.com/u/1721820?s=50" width="50" height="50">](https://github.com/pitrou)<a name="note_8"></a>|@pitrou set owner to @teratorn|
|-|-|

Fine now, +1.

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_9"></a>|@teratorn set status to `closed`|
|-|-|

(In [32962]) Merge rename-long-5332: rename variables named 'long' to avoid confusing 2to3
Author: teratorn
Reviewer: antoine
Fixes: [#5332](https://github.com/twisted/twisted/issues/5332)
